components/kyma-environment-broker/cmd/broker/provisioning_test.go

```diff
@@ -46,6 +46,34 @@ func TestProvisioning_HappyPath(t *testing.T) {
 	suite.AssertProvisioningRequest()
 }
 
+func TestProvisioning_Preview(t *testing.T) {
+	// given
+	suite := NewBrokerSuiteTest(t)
+	defer suite.TearDown()
+	iid := uuid.New().String()
+
+	// when
+	resp := suite.CallAPI("PUT", fmt.Sprintf("oauth/v2/service_instances/%s?accepts_incomplete=true", iid),
+		`{
+					"service_id": "47c9dcbf-ff30-448e-ab36-d3bad66ba281",
+					"plan_id": "683b232e-b5b3-4c81-8238-f6f9d8385de7",
+					"context": {
+						"globalaccount_id": "g-account-id",
+						"subaccount_id": "sub-id",
+						"user_id": "john.smith@email.com"
+					},
+					"parameters": {
+						"name": "testing-cluster"
+					}
+		}`)
+	opID := suite.DecodeOperationID(resp)
+
+	suite.processProvisioningByOperationID(opID)
+
+	suite.WaitForOperationState(opID, domain.Succeeded)
+	// todo: assert Kyma resource is created
```

Comment:
nit: is it planned as part of this PR or a later follow-up?

Reply:
it is for later PR when create kyma steop is done